artemis/values.yaml

```diff
@@ -137,6 +140,21 @@ artemis:
 
   affinity: {}
 
+  autoscaler:
+    # Wether the autoscaling should be used for the Artemis deployment
+    enabled: false
+    # IP address range that should be allowed to query metrics (must include the IP of the deployed prometheus server)
+    monitoringIp: "0.0.0.0/0"
```

Comment:
I guess the default value for this can't be limited as we don't know the IPs before, right?

Reply:
Or just allow no monitoring? Not sure how critical this is?